### PR TITLE
Fix a few more inconsistencies when loading and dumping JSON.

### PR DESCRIPTION
**Pull request by jezdez**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

These change have been added since c2429e92d2a5dd58cbe378b9cc06f3be969f747e.

## Related Tickets & Documents

Refs #2807. Original work in: #2817

### CODE REVIEW

File: redash/authentication/jwt_auth.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ def get_public_keys(url):
         if 'keys' in data:
             public_keys = []
             for key_dict in data['keys']:
-                public_key = jwt.algorithms.RSAAlgorithm.from_jwk(json.dumps(key_dict))
+                public_key = jwt.algorithms.RSAAlgorithm.from_jwk(json_dumps(key_dict))
```
Comment (arikfr) [inline redash/authentication/jwt_auth.py]: Not sure we should use `json_dumps` here.

Comment (jezdez) [reply redash/authentication/jwt_auth.py]: Why? `json_dumps` is just a pass through for with our JSONEncoder which won't get triggered with the payload here.

This is just a step to make sure we use `simplejson` everywhere and not the outdated Python 2.x stdlib library. Once we're on Python 3 we can revert the use of simplejson to the stdlib version of course.

Comment (jezdez) [reply redash/authentication/jwt_auth.py]: Just changing that to simplejson.dump instead.

**Comment by arikfr** [comment]
Thanks.

**Pull request merged**